.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -165,6 +165,18 @@ steps:
     name: "validatePods"
     displayName: "Validate Pods"
 
+  - script: |
```

Comment:
Should we add cilium agent restarts as well ?

Reply:
I think that check can be reserved for cilium nightly pipeline runs